oracle/oracle_config.ini

```diff
@@ -1,2 +1,4 @@
 [delphi client]
-base-url=localhost:5000
\ No newline at end of file
+base-url=localhost:5000
```

Comment:
is this the url that's used, or the url that's in Config.cs?

Reply:
Config.cs uses that url

Reply:
So which url is being used where?

Reply:
This url is being used in DelphiClient

Reply:
What does Config.cs use it's url for? Is it the same as what's being done in DelphiClient?